### DISCUSSION TITLE
The airflow interface log is always loading, but the log interface has data

### DISCUSSION THREAD

**Discussion by leoArtefact** [apache/airflow#26062] [Q&A]
![image](https://user-images.githubusercontent.com/101545234/187428111-21ded6c0-b36e-4fc2-b24d-fc815c1d3668.png)
![image](https://user-images.githubusercontent.com/101545234/187428605-be89d88e-8451-4b15-9039-355cefbb4a8a.png)

 all the url result are 200.  and log interface is correct , but the web is always loading ,I can't know why,
please help me solve this problem ,thanks very much

**Comment by leoArtefact** [comment]
pls help me

**Comment by leoArtefact** [comment]
the version is 2.2.3

**Reply by potiuk** [reply]
I think it has been fixed in newer versions. Please upgrade (there are likely 100s of other problems solved in versions since 2.2.3 so you should upgrade anyway (and if there will be any fix it willl only be applied in the latest version, so you will have to upgrade anyway.

**Reply by leoArtefact** [reply]
Thanks,i'll try
